spec/views/posts/index.html.erb_spec.rb

```diff
@@ -1,9 +1,54 @@
 require 'rails_helper'
 
-RSpec.describe 'posts/index.html.erb', type: :view do
-  it 'displays a list of posts' do
-    render
-    expect(rendered).to include('<h1>List of Posts</h1>')
-    expect(rendered).to include('<p>Find me in app/views/posts/index.html.erb</p>')
+RSpec.describe 'User post index page', type: :system do
+  before(:each) do
+    @user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+    @post = Post.create(author: @user, title: 'Hello', text: 'This is my first post')
+
+    6.times do |i|
+      Comment.create(post: @post, user: @user, text: "Comment #{i}")
+    end
+  end
+
+  after(:each) do
+    Comment.destroy_all
+    Post.destroy_all
+    User.destroy_all
+  end
+
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=post%20index%20page%3A-,I%20can%20see%20the%20user%27s%20profile%20picture.,-I%20can%20see), Please add a test that asserts you can see the user's profile picture.